README.md

```diff
@@ -1,3 +1,6 @@
 # githubtest
```

Comment:
This appears to be an additional line added to the README.md file, however, it is not very clear what the intended message is. It mentions a "testd branch daily", but it is not clear what this means or what the purpose of this branch is. It is important to provide clear and concise information in commit messages to help others better understand the changes made to a repository.